iot-e2e-tests/common/pom.xml

```diff
@@ -44,6 +44,30 @@
             <artifactId>iot-service-client</artifactId>
             <version>1.16.0</version>
         </dependency>
+        <dependency>
+            <groupId>com.microsoft.azure.sdk.iot.provisioning</groupId>
+            <artifactId>provisioning-device-client</artifactId>
+            <version>1.4.0</version>
```

Comment:
>. [](start = 22, length = 1)

we will need to change our release scripts to update versions...maybe create a task for it?

Reply:
I'll take care of this now, good catch